src/docs/background/payment-methods-overview/index.md

```diff
@@ -0,0 +1,141 @@
+the transaction process to allow the provider to accept or decline the
+transaction. If the provider declines the transaction, Stripe collects
+no processing fees since money has not been transferred.
+
+Finally, the transaction process can hold money in Stripe until an
+explicit payout (The hold should not exceed three months). The mechanism
+allows you to build a transaction process where the money is in an
+escrow-like hold until an explicit payout. The hold ensures some safety
+for the customer as the marketplace controls the money until a payout.
+The process can issue the payout in a separate transition.
+
+### Push payment flow
+
+For push payments, there is no preauthorization. Once the customer
+confirms the payment, a charge is made immediately. The charge moves
+money immediately to the provider's Stripe Connect account.
+
+![Push payment flow](push-payment-flow.png 'Push payment flow in Flex.')
+
+This means that refunds can be issued only at a point where money has
+already moved and the marketplace must cover Stripe fees. You should
+take this into account when designing the transaction process of your
+marketplace. Either declining the transaction should be disabled,
+handled through availability management, or accounted for in
+commissions.
+
+The rest of the money flow is the same as with card payments and has the
+same features and capabilities.
+
+## Using different payment methods in your marketplace
+
+### Customizing the transaction process
+
+The
+[default transaction process](https://github.com/sharetribe/flex-example-processes/tree/master/flex-default-process)
+in Flex supports card payments. The general article on
+[the transaction process](/background/transaction-process) describes the
+process in more detail.
+
+> **A note on Google Pay, Apple Pay, and Microsoft Pay**
+>
+> Even though Google Pay, Apple Pay, and Microsoft Pay are similar to
+> card payments, they require some changes to the default implementation
+> of FTW. To enable them in FTW, you need to follow the
+> [Stripe instructions on the Request Payment Button](https://stripe.com/docs/stripe-js/elements/payment-request-button).
+
+If you wish to enable push payments, you need to adapt your transaction
+process. For instance, you need to add a new transition that includes
+the
+[stripe-create-payment-intent-push](/references/transaction-process-actions/#actionstripe-create-payment-intent-push)
+action. Further, because push payments do not have a preauthorization
+phase, it is recommended to avoid that in the transaction process and
+use an _instant booking_ type of flow. The example below describes the
+minimum recommended changes in the two transitions:
+`request-push-payment` and `confirm-payment-instant-booking`. The example
+illustrates how you can still use the preauthorization step for card
+payments.
+
+![Push payment process](push-payment-process.png 'Push payment process example.')
+
+### Handling payment methods in the UI
```

Comment:
```suggestion
### Handling payment methods in the client app
```

part of this happens in client app's server, part in UI layer

Reply:
@Gnito which part is happening in the server?